analyzers/src/SonarAnalyzer.Common/Rules/GenericInheritanceShouldNotBeRecursiveBase.cs

```diff
@@ -21,6 +21,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using Microsoft.CodeAnalysis;
+using SonarAnalyzer.Extensions;
```

Comment:
`MethodsShouldNotHaveIdenticalImplementationsBase` should be also updated to remove `IsExcludedFromBeingExamined`

Reply:
Here, the check is more than just the same is redundant positional record check, it plays a role also for checking namespaces for top level statements.